README.md

```diff
@@ -1,5 +1,168 @@
+-->
+
+## FAQ
+
+<details><summary><b>What games support it?</b></summary><br>
+
+You scrolled too fast! [Check out the growing list of games](#show-me-the-games) that offer Playdate Achievements above.
+
+</details>
+
+<details><summary><b>How does it work?</b></summary><br>
+
+Playdate Achievements takes advantage of the `Shared/` folder that Playdate makes available to all games. By establishing an open standard for achievements, games can record available and earned achievements in dedicated subfolders which any other game or viewer app can read. This enables apps like [Trophy Case](https://github.com/gurtt/trophy-case) to present an aggregated list of all achievements earned across games.
+
+</details>
+
+<details><summary><b>Does it support non-Catalog games?</b></summary><br>
+
+Yes! Playdate Achievements is an open standard available to _all_ games regardless of whether they’ve been accepted to Catalog or their distribution method. Games hosted on [itch.io]() or your own site are absolutely welcome and fully supported.
+
+</details>
+
+<details><summary><b>Does it work with games made in Pulp?</b></summary><br>
+
+[Pulp](https://play.date/pulp/) games don't have access to the `Shared/` folder or Lua code libraries, and therefore aren’t compatible with Playdate Achievements. However, you may be able to integrate the library after using [Pulp Mill](https://github.com/nstbayless/pulp-to-lua), a Pulp-to-Lua transpiler.
+
+</details>
+
+<details><summary><b>How can I add Playdate Achievements to my game?</b></summary><br>
+
+Check out our [getting started guide](https://github.com/PlaydateSquad/pd-achievements/blob/main/README.md) in the [`pd-achievements`](https://github.com/PlaydateSquad/pd-achievements) repo on GitHub.
+
+</details>
+
+<details><summary><b>Can I add my game to this page?</b></summary><br>
+
+All games that integrate Playdate Achievements are welcome! [Open a pull request](https://github.com/PlaydateSquad/pd-achievements/compare) to add your game to the list and a member of the community will review and merge the changes.
+
+</details>
+
+<details><summary><b>How do I report a bug?</b></summary><br>
+
+All framework development occurs in the [`pd-achievements`](https://github.com/PlaydateSquad/pd-achievements) repo on GitHub. Please open an issue to report any bugs you find. If you find bugs in the Trophy Case viewer app, report them in the separate [`trophy-case`](https://github.com/gurtt/trophy-case) repo.
+
+</details>
+
+<details><summary><b>I already implemented my own achievements system — what should I do?</b></summary><br>
```

Comment:
The pickiest of nits: there shouldn't be spaces around an em dash.

Reply:
I support the picking of typographic nits.